src/components/LoansByCategory/CategoryRowHover.vue

```diff
@@ -19,7 +19,7 @@
 						{{ cleanName }}
 					</template>
 				</h2>
-				<p v-if="showCategoryDescription" class="category-description show-for-large">
+				<p class="category-description show-for-large">
```

Comment:
I disabled this `v-if` since even if we don't have a description, we need a min height to allow clicking on the row title. This change is only for rows containing hover cards.

Reply:
Should be ok, may want to verify no errors are thrown on something without a description. If so you could throw a check in where we try to use "loanChannel.description"